src/main/java/io/jsonwebtoken/impl/DefaultJwtParser.java

```diff
@@ -58,6 +62,54 @@
 
     private SigningKeyResolver signingKeyResolver;
 
+    Claims expectedClaims = new DefaultClaims();
+
+    @Override
+    public JwtParser expectIssuedAt(Date issuedAt) {
+        if (issuedAt != null) {
```

Comment:
no null check - just invoke the delegate directly (and null values are not allowed for required claim checks as indicated in the delegate method comments below).